getting-started/modules/zgw/portaaltaak.md

```diff
@@ -0,0 +1,28 @@
+# Notificaties API
+
```

Comment:
Maybe add a description like:
This plugin makes it possible for Valtimo to communicate and interact with a Portal task specification compliant ZGW component.

Reply:
Are you sure? I don't see the description part in other modules.md as well.